source/_patterns/01-atoms/dropdown/_dropdown.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../00-base/config';
```

Comment:
We might need to account for these kind of changes depending on the timing of @tjheffner cleanup branch.